### DISCUSSION TITLE
Integrating Grocy and Barcode buddy as Add-ons within Home Assistant

### DISCUSSION THREAD

**Discussion by dvpfig** [Forceu/barcodebuddy#223] [Q&A]
Hi, 

Not sure how up-to-date this repository is kept with the changes in HA, but the ingress type of access in HA add-ons seems not to be supported by Barcode buddy. Or at least I cannot find a way. 

I have both Grocy and Barcode buddy as two Add-ons in the same HassOS instance. 
Grocy is well installed and communicates with my Grocy on Android app, so it's reachable. 

For the Barcode buddy add-on, when trying to setup the connection to Grocy. I don't manage to make it work. 

The screen requests a IP address for Grocy and /api/ at the end. However, with ingress, I cannot seem to make Grocy accessible via an IP and port number. 

Is there a way with latest versions of HA, Grocy, to make Barcode buddy work?

**Comment by Tim-G-Winters** [comment]
I too am having trouble connecting barcode buddy to Grocy. I cant get the url or the API key to work.
